static/assets/datasets/chargen/dataset-pt-br.json

```diff
@@ -602,11 +602,7 @@
 			"key": "elf",
 			"name": "Elfo",
 			"weight": 2,
-			"age": [
-				[20, 40],
-				[41, 200],
-				[201, 400]
-			],
+			"age": [[0, 0]],
```

Comment:
why?

Reply:
Because this is what is used the default english dataset, I assume those values were copied and pasted wrongly...